services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/_dependencies.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from typing import Any, Callable
+
+from fastapi import Request
+from starlette.datastructures import URL
+
+logger = logging.getLogger(__name__)
+
+
+#
+# DEPENDENCIES
+#
+
+
+def get_reverse_url_mapper(request: Request) -> Callable:
```

Comment:
MINOR: thisone we might want to put it somewhere in `servicelib.fastapi` as utils since we are using it everywhere! :-)

Reply:
actually I need to check if it is used here...